tests/browserstack/specs/ballotMainTest.js

```diff
@@ -108,85 +108,134 @@ describe('Basic cross-platform We Vote test',  () => {
 
     // //////////////////////
     // Change Address
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('changeAddressHeaderBar'); // Open the "Change Address" modal
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('profileCloseSelectBallotModal');
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-
-    // //////////////////////
-    // We want to start by setting the location, which will automatically choose the next upcoming election for that address
-    await simpleClick('changeAddressHeaderBar'); // Opens the "Enter Your Full Address" link
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-
-    if (isIOS && isCordovaFromAppStore) {
-      await setNewAddressIOS('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
-    } else if (isAndroid && isCordovaFromAppStore) {
-      await setNewAddressAndroid('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
-    } else {
+    if (isDesktopScreenSize) {  // Desktop/tablet version
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      await simpleClick('changeAddressOrElectionHeaderBar'); // Open the "Change Address" modal
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
       await setNewAddress('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
+      await simpleClick('changeAddressOrElectionHeaderBar'); // Open the "Change Address" modal
+    } else {  //  Mobile version
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      await simpleClick('changeAddressHeaderBar'); // Opens the "Enter Your Full Address" link
+      if (isIOS && isCordovaFromAppStore) {
+        await setNewAddressIOS('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
+      } else {
+        await setNewAddressAndroid('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
+      }
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      await simpleClick('ballotTitleHeaderSelectBallotModal'); // Opens the Enter Your Full Address link
     }
-    await browser.pause(PAUSE_DURATION_BALLOT_LOAD);
+
```

Comment:
We found that we needed a longer pause to allow the ballot to load, so this is still needed:
`await browser.pause(PAUSE_DURATION_BALLOT_LOAD);`